### ISSUE TITLE
physical meanings of beta and sigma in smearing SCF

### ISSUE THREAD

**Issue by JunboLu** [pyscf/pyscf#2571]
Hello,
  I am using smearing SCF as:
beta = 1
mf_ft = addons.smearing(mf, sigma=1./beta, method='fermi', fix_spin=True)
mf_ft.kernel()
  What do beta and sigma stand for? And what is the unit of beta?
  With my best regards,
  Junbo

**Comment by tberkel** [comment]
sigma is the electronic temperature (technically kT) in units of Hartree.
In the example you found, beta is simply being used in its conventional
definition of beta = 1/kT.

**Comment by JunboLu** [comment]
Thank you!
Is there recommended value for beta? I calculate EuCl3 molecule, and its electronic configuration is 4f^6.

**Comment by tberkel** [comment]
Not really, it depends a bit on why you are using smearing. If it's just to
facilitate SCF convergence, then you probably want the smallest sigma
possible that still gives converged SCF.

**Comment by sunqm** [comment]
Please note that if sigma is too small, like 1e-5, you may encounter convergence issues due to numerical noise in the orbital occupancies. To avoid these issues, I recommend sigma not smaller than 1e-3
